english/find-exit/into-forest/forest.md

```diff
@@ -17,4 +17,16 @@ Version control rules
 
 Managed projects is the aim
 
-Peace of mind the gain.
\ No newline at end of file
+Peace of mind the gain.
+
+But only for a short while
+
+That something watching you emerges from the shadows
+
+It's a cougar, stalking it's prey
+
+It approaches you, and you can do nothing but wait for it to make it's move
+
+Then it asks "What's your name?"
+
+[Do you talk to it?] ()
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.